### DISCUSSION TITLE
Picklist and Drag & Drop documentation enhancement

### DISCUSSION THREAD

**Discussion by JIBIN-P** [primefaces/community#733] [PrimeNG] [answered]
For Picklist the only import statement is for the PickList Module which is mentioned in the document but It doesn't specify the dependency on the Drag&Drop module which I think should be added in the documentation. and for Drag & Drop, there is a dependency on the @angular/cdk which also should be mentioned in the doc but as of now there is no reference to it, For a beginner it can be difficult to understand why they're getting the error when they're following the example given on the page. Please let me know if they're appropriate points or not, As per accordingly I will try to open a PR for it.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for the feedback we've updated the documentation https://github.com/primefaces/primeng/commit/55cbc09331f897c00fe168410cc9c214247033de
